.scripts/languages.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+function run_commands($commands) {
+	foreach ($commands as $command) {
+		echo "$command\n";
+		passthru($command, $return_val);
+		if ($return_val !== 0) {
+			echo "Error executing command! Interrupting!\n";
+			exit(2);
+		}
+	}
+}
+
+$elgg_path = dirname(__DIR__);
+
+// Setup. Version checks are here so we fail early if any deps are missing
+run_commands([
+	"tx --version",
+	"cd $elgg_path",
+	"tx pull -af --minimum-perc=95",
+]);
+
+// Clean translations
+$cleaner = new Elgg\I18n\ReleaseCleaner();
+$cleaner->cleanInstallation(dirname(__DIR__));
+foreach ($cleaner->log as $msg) {
+	echo "ReleaseCleaner: $msg\n";
+}
+
+run_commands([
+	"git add .",
+	"git commit -am \"chore(i18n): update translations\"",
```

Comment:
This should open a new branch and afterwards tell the user to push to their fork.